databook/physiology/ophys/ophys-background.md

```diff
@@ -1,6 +1,6 @@
 # Ophys background
```

Comment:
again, not ophys background. We want `Ophys`

Reply:
revised